crates/extensions/tedge_mqtt_bridge/src/lib.rs

```diff
@@ -57,6 +57,8 @@ use crate::topics::matches_ignore_dollar_prefix;
 use crate::topics::TopicConverter;
 pub use config::*;
 
+const MAX_PACKET_SIZE: usize = 10 * 1024 * 1024;
```

Comment:
Looks like we have some inconsistency with packet sizes used across the project.
1. In `mqtt_channel::config::Config`: 1024 * 1024
2. In `bridge::common_mosquitto_config::CommonMosquittoConfig`: 256 * 1024 * 1024 - 1
3. In `tedge::cli::mqtt::MAX_PACKET_SIZE`: 10 * 1024 * 1024

Any reason why you chose the 3rd option and not the second one, keeping it consistent with the existing bridge config?

Reply:
I didn't realise that mosquitto used a completely different name for the configuration, so didn't notice it. I'll change this to be consistent with the existing bridge.